backstop_data/engine_scripts/puppet/search/find-activities-delete-modal.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = async (engine, scenario, vp) => {
+  await require('./find-activities')(engine, scenario, vp);
+  await engine.click('.selector tbody tr[id*="rowid"]:nth-child(2) a[title="Delete Activity"]');
```

Comment:
isn't there a better selector? This one seems too specific (especially with the `nth-child()` bit)

Reply:
Nope! For this particular form the table HTML is inconsistent than others.

Reply:
However I think we can remove [id*="rowid"] since we are already using nth child selector. Let me update it.